### PR TITLE
Updates description for AI core displays and makes them buildable

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

Also adds follow buttons to both the display core and data core.
Makes the AI core displays buildable

# Wiki Documentation

# Changelog

:cl:  
rscadd: The AI Display Core now mentions that it doesn't actually hold the AI. In addition ghosts can examine it to see laws of AI's for easy access.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/decentralized/ai_data_core.dm [hunk 1]
```diff
@@ -59,13 +59,14 @@ GLOBAL_VAR_INIT(primary_data_core, null)
...
 
+//NOTE: See /obj/machinery/status_display/examine in ai_core_display.dm
 /obj/machinery/ai/data_core/examine(mob/user)
 	. = ..()
 	if(!isobserver(user))
 		return
 	. += "<b>Networked AI Laws:</b>"
 	for(var/mob/living/silicon/ai/AI in GLOB.ai_list)
-		var/active_status = ""
+		var/active_status = "(Core: [FOLLOW_LINK(user, AI.loc)], Eye: [FOLLOW_LINK(user, AI.eye)])"
```
Comment (Bibby0110) [inline code/modules/mob/living/silicon/ai/decentralized/ai_data_core.dm]:
```suggestion
		var/active_status = "(Core: [FOLLOW_LINK(user, AI.loc)], Eye: [FOLLOW_LINK(user, AI.eyeobj)])"
```

File: code/modules/mob/living/silicon/ai/decentralized/ai_core_display.dm [hunk 2]
```diff
@@ -19,6 +21,24 @@
...
 
+//NOTE: See /obj/machinery/ai/data_core/examine in ai_data_core.dm
+/obj/machinery/status_display/examine(mob/user)
+	. = ..()
+	if(!isobserver(user))
+		return
+	. += "<b>NOTE: Does not contain the AI. Look for AI Data Cores to find the *actual* location of the AI</b>"
+	. += "<b>Networked AI Laws:</b>"
+	for(var/mob/living/silicon/ai/AI in GLOB.ai_list)
+		var/active_status = "(Core: [FOLLOW_LINK(user, AI.loc)], Eye: [FOLLOW_LINK(user, AI.eye)])"
```
Comment (Bibby0110) [inline code/modules/mob/living/silicon/ai/decentralized/ai_core_display.dm]:
```suggestion
		var/active_status = "(Core: [FOLLOW_LINK(user, AI.loc)], Eye: [FOLLOW_LINK(user, AI.eyeobj)])"
```

**Comment by Bibby0110** [comment]
easy

**Comment by Moja10** [comment]
Best maintainer of all time????

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**